docs/dev-guide.md

```diff
@@ -0,0 +1,161 @@
+## Getting Mainflux
+
+Mainflux can be fetched from official [Mainflux GitHub repository](https://github.com/Mainflux/mainflux):
+
+```
+mkdir -p $GOPATH/src/github.com/mainflux
+cd $GOPATH/src/github.com/mainflux
+git clone https://github.com/mainflux/mainflux.git
+cd mainflux
+```
+
+## Building
+
+### Prerequisites
+Make sure that you have [Protocol Buffers](https://developers.google.com/protocol-buffers/) compiler (`protoc`) installed.
```

Comment:
I think that `protoc` version matters and that we currently use `3.5.1`. Maybe it's a good idea to mention that here.

Reply:
I would leave this out of the generic doc, as it is just the implementation detail.